src/Bridges/detbridge.jl

```diff
@@ -134,9 +139,10 @@ end
 function MOI.get(model::MOI.ModelLike, a::MOI.ConstraintPrimal, c::LogDetBridge)
     d = length(c.lcindex)
     Δ = MOI.get(model, MOI.VariablePrimal(), c.Δ)
-    t = MOI.get(model, MOI.ConstraintPrimal(), c.tlindex) - sum(log.(Δ[trimap.(1:d, 1:d)]))
+    t = MOIU.evalvariables(vi -> MOI.get(model, MOI.VariablePrimal(), vi), c.t)
```

Comment:
Why not do compute it `t` from the `ConstraintPrimal` of `tlindex` and the `ConstraintPrimal` and the values of the `x`s in `lcindex`s ?
The value of `u` can be computed from one of the `ConstraintPrimal` of the `lcindex`.
This would allow to avoid storing `t` and `u` in the bridge.

Reply:
This part caused me the most confusion. I think I don't fully understand the purpose of getting constraint primals. Could you update the PR with your suggestion? I'll address your other comments today.